app/main.py

```diff
@@ -1,25 +1,69 @@
+from __future__ import annotations
+from typing import Any
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Visitor, name: str) -> None:
+        self.name = "_" + name
+
+    def __get__(
+            self,
+            instance: Any,
+            owner: Visitor
+    ) -> Visitor:
+        return getattr(instance, self.name)
+
+    def __set__(self, instance: SlideLimitationValidator, value: int) -> None:
+        if value not in range(self.min_amount, self.max_amount + 1):
+            raise ValueError("Sorry the visitor can't use the slide")
```

Comment:
The error message could be more generic without linking to a specific visitor, e.g.
```suggestion
            raise ValueError(f"{value} is not within range [{self.min_count}, {self.max_count}]")
```